.github/workflows/dotnetcore.yml

```diff
@@ -29,19 +28,11 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4.1.1
-
-    - name: Setup .NET 6.0.x
-      uses: actions/setup-dotnet@v4.0.0
-      with:
-        dotnet-version: 6.0.x
-    - name: Setup .NET 8.0.x
-      uses: actions/setup-dotnet@v4.0.0
-      with:
-        dotnet-version: 8.0.x
-    - name: Setup .NET 9.0.x
+        
+    - name: Setup .NET 9.x
       uses: actions/setup-dotnet@v4.0.0
       with:
-        dotnet-version: 9.0.100-rc.2.24474.11
+        dotnet-version: 9.x
 
     - name: Run the tests
       run: dotnet test Wilson.sln --collect:"XPlat Code Coverage" --settings:./build/CodeCoverage.runsettings
```

Comment:
To fix #3001 update the line a few lines below:
`dotnet tool install -g dotnet-reportgenerator-globaltool --version 5.4.0`